### PR TITLE
Rebuild/data model

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

**Comment by leggsimon** [comment]
LGTM

File: getFrontPageData.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	body: JSON.stringify({
+		query,
+		variables
+	}),
+	headers: {
+		'Content-Type': 'application/json'
+	}
+})
+	.then(response => response.json())
+	.then(data => console.log(JSON.stringify(data, null, 2)));
```
Comment (leggsimon) [inline getFrontPageData.js]: Do we want this to `console.log`?

Comment (aintgoin2goa) [inline getFrontPageData.js]: Yeah - this is a script that gets the fixture used in the test - is run from `make fixturez

Comment (leggsimon) [inline getFrontPageData.js]: Ok coolz

**Pull request merged**